google-userinfo/background/authorize.js

```diff
@@ -0,0 +1,74 @@
+const REDIRECT_URL = browser.identity.getRedirectURL();
+const CLIENT_ID = "YOUR-CLIENT-ID";
+const SCOPES = ["openid", "email", "profile"];
+const AUTH_URL =
+`https://accounts.google.com/o/oauth2/auth
+?client_id=${CLIENT_ID}
+&response_type=token
+&redirect_uri=${encodeURIComponent(REDIRECT_URL)}
+&scope=${encodeURIComponent(SCOPES.join(' '))}`;
+const VALIDATION_BASE_URL="https://www.googleapis.com/oauth2/v3/tokeninfo";
+
+function extractAccessToken(redirectUri) {
+  let m = redirectUri.match(/[#\?](.*)/);
+  if (!m || m.length < 1)
+    return null;
+  const params = new URLSearchParams(m[1].split("#")[0]);
```

Comment:
I'm probably old school, and it is just a nit.  const tells me something about the variable, that it is being protected from change, for some reason.  In this case the variable is in a scope that cant be touched and the variable is dropped right away.  So to me, seeing const there makes my brain spin on "why".